zeebe/clients/java/src/test/java/io/camunda/zeebe/client/TopologyRequestRestTest.java

```diff
@@ -44,47 +45,57 @@ public final class TopologyRequestRestTest extends ClientRestTest {
   void shouldRequestTopology() throws ExecutionException, InterruptedException {
     // given
     gatewayService.onTopologyRequest(
-        2,
-        10,
-        3,
-        "1.22.3-SNAPSHOT",
-        broker(
-            0,
-            "host1",
-            123,
-            "1.22.3-SNAPSHOT",
-            partition(0, LEADER, HEALTHY),
-            partition(1, FOLLOWER, UNHEALTHY)),
-        broker(
-            1,
-            "host2",
-            212,
-            "2.22.3-SNAPSHOT",
-            partition(0, FOLLOWER, HEALTHY),
-            partition(1, LEADER, HEALTHY)),
-        broker(
-            2,
-            "host3",
-            432,
-            "3.22.3-SNAPSHOT",
-            partition(0, FOLLOWER, UNHEALTHY),
-            partition(1, INACTIVE, UNHEALTHY)));
+        new TopologyResponse()
+            .gatewayVersion("1.22.3-SNAPSHOT")
+            .clusterSize(3)
+            .replicationFactor(2)
```

Comment:
💭 Replication factor 2 is uncommon. As an example test, replication factor 3 would be a better fit. This is also already represented in the data below: each partition has 3 replicas (one on each broker)

Reply:
Good catch! 👍